src/main/java/seedu/duke/command/ArchiveNoteCommand.java

```diff
@@ -47,31 +51,27 @@ public ArchiveNoteCommand(String title) {
 
     @Override
     public String execute() {
-        boolean isArchived;
-        String archivedTitle = "";
+        String archivedTitle;
```

Comment:
No need to have this variable here. If title is blank, you can reuse the variable "title" and assign the title here. At the end you will only need 1 return Formatter.formatString(ARCHIVE_NOTE_MESSAGE + title);